### DISCUSSION TITLE
How are numpad buttons named?

### DISCUSSION THREAD

**Discussion by Sibseven** [sezanzeb/input-remapper#855] [Q&A]
I tried KEY_0 and KEY_NUMERIC_0, but they didnt work. Different online key-testers recognized KEY_NUMERIC_0 as left-click. Remapping arrows did work.

**Comment by sabudum** [comment]
KEY_KP0, KEY_KP1, KEY_KP2, etc...
The same goes for math signs from the numpad:
KEY_KPPLUS, KEY_KPMINUS, KEY_KPASTERISK, etc...

You can get a full list of all button names with the command:
input-remapper-control --symbol-names

**Reply by Sibseven** [reply]
Thanks, will try

**Reply by Renari** [reply]
Thanks, this did the trick I thought it was `KEY_NUMERIC_0` but that wasn't the same.
